ginga/qtw/Widgets.py

```diff
@@ -1580,6 +1597,18 @@ def add_widget(self, child, row, col, stretch=0):
         self.widget.layout().addWidget(w, row, col)
         self.make_callback('widget-added', child)
 
+    def insert_cell(self, row, col):
```

Comment:
See comment about ``insert_cell()`` below...

Reply:
above?

Reply:
I think it depends on the view. In diff, it is ordered by the files, but in the main area, it is ordered by comment timestamp. I think you mean this comment: https://github.com/ejeschke/ginga/pull/968/files#r679653666 ?